procurify-dashboard/index.html

```diff
@@ -0,0 +1,317 @@
+                        <div class="user-info activity-order-info col">
+                      <p class="sub-info activity-date">Aug 10, 2018, 12:45 PM</p>
+                    </div>
+                    </div>
+                    <div class="activity-heading row">
+                    <div class="col">
+                    <div class="user-info activity-status-wrap">
+                      <p class="activity-status">Item Purchased</p>
+                      <p class="sub-info">from <span>Order #113</span></p>
+                    </div>
+                  </div>
+                    </div>
+                  <!-- <div class="activity">
+                    <p class="activity-full">Croissant danish candy canes bear claw dragée pie gingerbread. Dragée jelly beans jelly pudding bonbon. Chocolate apple pie toffee cookie bonbon oat cake halvah. Gummies marzipan muffin candy pastry chocolate bar.</p>
+                  </div> -->
+            </div>
+            <div>
+              <div class="budgets-overview-wrap">
+                <div class="budgets-overview">
+                    <div class="panel">
+                        <div class="">
+                          <div class="panel-title">Budgets Overview</div>
+                        </div>
+                        <div class="field">
+                            <label for="">Location</label>
+                            <div class="styled-select">
+                            <select class="budget-dropdown" name="" id="">
+                              <option value="Select A Location">Select a Location</option>
+                            </select>
+                            </div>
+                          </div>
+                    </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-5">
+          <div class="panel comments-panel">
+            <div class="comments-content">
+              <div class="panel-title">Recent Comments</div>
+              <div class="comments-container">
+                <div class="comment-wrap">
+                <a class="object">
+                    <div class="object-icon">
+                      <img src="https://mir-s3-cdn-cf.behance.net/project_modules/1400/468b3459859965.5a6511e582352.jpg" alt="">
+                    </div>
+                  </a>
+                  <div class="comment-block">
+                    <div class="comment">
+                      <h5>Sohn Jtamos</h5>
+                      <p class="sub-info sub-info">23 days ago <span>Order #113</span></p>
+                      <p class="comment-full">Croissant danish candy canes bear beans jelly pudding bonbon. Chocolate apple pie toffee cookie bonbon oat cake halvah. Gummies marzipan muffin candy pastry chocolate bar.</p>
+                    </div>
+                  </div>
+                </div>
+                <div class="comment-wrap">
+                <a class="object">
```

Comment:
Again, indentation here. It could be your editor doing it by accident when you copy and paste.